socorro/mozilla_rulesets.py

```diff
@@ -121,6 +122,7 @@
     MemoryReportExtraction(),
     # generate signature now that we've done all the processing it depends on
     SignatureGeneratorRule(),
+    SoftErrorsRule(),
```

Comment:
Judging from the comments, this list appears to be split in sections, and the new rule should go in the "post processing of the processed crash" section. I don't think signature generation will ever depend on the soft_errors field, but the section  structure makes sense, so we should stick to it.